api/solarperformanceinsight_api/models.py

```diff
@@ -316,7 +316,7 @@ class PVWattsInverterParameters(BaseModel):
     eta_inv_ref: float = Field(
         0.9637, description="Reference inverter efficiency, unitless"
     )
-    _modelchain_ac_model: str = PrivateAttr("pvwatts")
+    _modelchain_ac_model: str = PrivateAttr("pvwatts_multi")
```

Comment:
I think we need to use `pvwatts` if only 1 inverter is present. `pvlib.inverter.pvwatts_multi` assumes tuple/list input and will return the wrong result if provided with only a Series. Is that doable here or not trivial? I'm guessing the latter.

I'm open to changing things in pvlib, but it's also not trivial there.

Reply:
same for sandia_multi

Reply:
Do you mean calls to `ModelChain.run_model` always need input as a list (`ModelChain.run_model([array_0_data, array_1_data])`) for this to work? That is the current plan for calling run_model, even with a single array.

Reply:
> I think we need to use pvwatts if only 1 inverter is present. 

Agree. 

The pvlib implementation could be made flexible to cover single and multiple inverters, if that becomes necessary.

Reply:
From just looking at the code, I think `pvwatts_multi` might actually work with a single array within the `ModelChain` workflow. `test_modelchain.py::test_ac_models` asserts that `mc.results.ac` is a Series. This would not be the case if `pvwatts_multi` performed a sum over the single series.